content/docs/command-reference/import-url.md

```diff
@@ -56,27 +55,29 @@ source.
 
 DVC supports several types of (local or) remote locations (protocols):
 
-| Type     | Description                                         | `url` format                               |
-| -------- | --------------------------------------------------- | ------------------------------------------ |
-| `local`  | Local path                                          | `/path/to/local/data`                      |
-| `s3`     | Amazon S3                                           | `s3://mybucket/data`                       |
-| `azure`  | Microsoft Azure Blob Storage                        | `azure://my-container-name/path/to/data`   |
-| `gs`     | Google Cloud Storage                                | `gs://mybucket/data`                       |
-| `ssh`    | SSH server                                          | `ssh://user@example.com:/path/to/data`     |
-| `hdfs`   | HDFS to file (explanation below)                    | `hdfs://user@example.com/path/to/data.csv` |
-| `http`   | HTTP to file with _strong ETag_ (explanation below) | `https://example.com/path/to/data.csv`     |
-| `remote` | Remote path (see explanation below)                 | `remote://myremote/path/to/data`           |
+| Type     | Description                       | `url` format example                                   |
+| -------- | --------------------------------- | ------------------------------------------------------ |
+| `s3`     | Amazon S3                         | `s3://bucket/key/to/data`                              |
+| `azure`  | Microsoft Azure Blob Storage      | `azure://container/path/to/data`                       |
+| `gdrive` | Google Drive                      | `gdrive://<folder-id>/data`                            |
+| `gs`     | Google Cloud Storage              | `gs://bucket/path/to/data`                             |
+| `ssh`    | SSH server                        | `ssh://user@example.com:/path/to/data`                 |
+| `hdfs`   | HDFS to file\*                    | `hdfs://user@example.com/path/to/data.csv`             |
+| `http`   | HTTP to file with _strong ETag_\* | `https://example.com/path/to/data.csv`                 |
+| `webdav` | WebDav to file\*                  | `webdavs://example.com/public.php/webdav/path/to/data` |
+| `local`  | Local path                        | `/path/to/local/data`                                  |
+| `remote` | Remote path\*                     | `remote://remote-name/path/to/data`                    |
```

Comment:
Same Qs. Added GDrive and WebDAV so far.

Reply:
it's hard to review this ... it's hard to understand what is changed exactly? could you summarize it please?